docs/light/getting_started.md

```diff
@@ -1,18 +1,26 @@
 # Getting Started
 
 To start a rest server, we need to specify the following parameters:
-| Parameter   | Type      | Default                 | Required | Description                                          |
-| ----------- | --------- | ----------------------- | -------- | ---------------------------------------------------- |
-| chain-id    | string    | null                    | true     | chain id of the full node to connect                 |
-| node        | URL       | "tcp://localhost:46657" | true     | address of the full node to connect                  |
-| laddr       | URL       | "tcp://localhost:1317"  | true     | address to run the rest server on                    |
-| trust-node  | bool      | "false"                 | true     | Whether this LCD is connected to a trusted full node |
-| trust-store | DIRECTORY | "$HOME/.lcd"            | false    | directory for save checkpoints and validator sets    |
 
-Sample command:
+| Parameter       | Type      | Default                 | Required | Description                                          |
+| -----------     | --------- | ----------------------- | -------- | ---------------------------------------------------- |
+| home            | string    | "$HOME/.gaiacli"        | false    | directory for save checkpoints and validator sets    |
+| chain-id        | string    | null                    | true     | chain id of the full node to connect                 |
+| node-list       | URL       | "tcp://localhost:26657" | false    | addresses of the full node to connect                |
+| laddr           | URL       | "tcp://localhost:1317"  | false    | address to run the rest server on                    |
+| trust-node      | bool      | "false"                 | false    | Whether this LCD trust full nodes or not             |
+| swagger-host-ip | string    | "localhost"             | false    | The IP of the server which Cosmos-LCD is running on  |
+| modules         | string    | "general,key,token"     | false    | enabled modules.                                     |
+
+* When the connected full node is trusted, then the proof is not necessary, so you can run Cosmos-LCD with trust-node option:
+```
+gaiacli advanced rest-server-swagger --chain-id {your chain id} --trust-node
```

Comment:
note advanced prefix will soon be removed https://github.com/cosmos/cosmos-sdk/pull/2014